independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1187 @@
+          },
+          {
+            "name": "app_key",
+            "in": "header",
+            "description": "Your API app key",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "App Key",
+            "x-data-threescale-name": "app_keys"
+          },
+          {
+            "name": "ResourceVersion",
+            "in": "header",
+            "description": "Resource version of the API",
+            "default": "v4",
+            "type": "string",
+            "required": true,
+            "x-ms-summary": "Resource Version"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "description": "Accept header",
+            "required": true,
+            "default": "application/json",
+            "type": "string",
+            "x-ms-visibility": "internal",
+            "x-ms-summary": "Accept"
+          },
+          {
+            "name": "scheduleDate",
+            "in": "query",
+            "description": "Scheduled date to get flights for. Format: yyyy-MM-dd. Defaults to today if not provided",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Schedule Date"
+          },
+          {
+            "name": "scheduleTime",
+            "in": "query",
+            "description": "Scheduled time to get flights from. Format: HH:mm",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Schedule Time"
+          },
+          {
+            "name": "flightName",
+            "in": "query",
+            "description": "Flight number as printed on the ticket",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Flight Name"
+          },
+          {
+            "name": "flightDirection",
+            "in": "query",
+            "description": "Direction of the flight",
+            "required": false,
+            "type": "string",
+            "enum": [ "A", "D" ],
+            "x-ms-summary": "Flight Direction"
+          },
+          {
+            "name": "airline",
+            "in": "query",
+            "description": "Prefix in flight number as printed on the ticket. For example: KL. This can either be a 2-character (IATA) airline prefix or 3-character (ICAO) airline prefix. Only one airline (or airlineCode) can be queried at the same time. ",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Airline"
+          },
+          {
+            "name": "airlineCode",
+            "in": "query",
+            "description": "NVLS code of an airliner.",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "Airline Code"
+          },
+          {
+            "name": "route",
+            "in": "query",
+            "description": "IATA or ICAO code of airport in route; multiple values, comma separated",
```

Comment:
@Miesjh please [add period](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#summary-and-description) at the end of description.